packages/site-admin/CHANGELOG.md

```diff
@@ -2,25 +2,27 @@
 
 Initial release of the site-admin package providing a framework for building modern WordPress admin interfaces.
 
-## Core Components
+### Components
 
 - `SidebarButton`: Base button component for sidebar interactions
 - `SidebarContent`: Main sidebar layout component with navigation context
 - `SidebarNavigationItem`: Interactive navigation items with icon support
 - `SidebarNavigationScreen`: Screen container for sidebar navigation
 - `SiteIcon`: Site icon display component
 
-## Routing System
+### Routing System
 - `RoutesContext`: Context provider for current matched route
 - `ConfigContext`: Context for router configuration and settings
 - `browserHistory`: Browser history instance for navigation state
 
-### Router Hooks
+#### Router Hooks
 - `useHistory`: Hook for navigation and history management
 - `useLink`: Hook for handling link navigation
 - `useLocation`: Hook for accessing current route location
 - `useMatch`: Hook for route matching functionality
 
 ## Next
 
+### Components
 - `SiteHub`: Site navigation and context switcher
+- `Link`: Router component providing declarative navigation
```

Comment:
We don't have one now, but I'm thinking we may want to have some kind of `Link` component in broader Calypso. Could we maybe rename this to `SiteLink` or something, to avoid potential naming collisions?

Reply:
If we stick to the idea that Calypso is just used as a place to hold the code, I think it’s fine to keep the Link component name. In case of name collisions, can we always use an alias?<br><br>otherwise, I’d go with something like `<NavLink />` or something that clearly refers to the navigation link concept.

Reply:
Not specifically a Calypso problem, because `Link` components are very common in frameworks/apps. I guess my question is, will this site-admin (and thus the site-admin `Link`) never be a mere subsection of a broader app, where let's say you want to put a Next.js `Link` and a site-admin `Link` in the same JS file? (The answer could be "no", just asking 😄)

Reply:
Fair question.
I’m not sure what name we could use instead, though.  `NavLink,` for instance, feels confusing to me since it sounds like it's tied to a navigation section. Could `RouterLink` work?

Reply:
If it's a "sub-router" specifically for site-admin, then maybe it should be called `SiteAdminLink`? And if it's a more general-use router, should it live somewhere else where it can be called something more generic? Or is there already an existing router we can reuse here?

Non-blocking, just some architecture thoughts.